demisto_sdk/commands/common/tools.py

```diff
@@ -2911,7 +2911,7 @@ def get_invalid_incident_fields_from_mapper(
             if simple := inc_info.get('simple'):
                 if '.' in simple:
                     simple = simple.split('.')[0]
-                if simple not in content_fields:
+                if simple not in content_fields and simple.lower() not in content_fields:
```

Comment:
content_fields are already lower arent they? Whats the point of the first check?

Reply:
or if it isnt lowercase,
 `if any(simple.lower() == field.lower() for field in content_fields)`

Reply:
no, only some of them are lower (build in fields), those in the ID set can be upper case as well. 

see this line:
```
content_incident_fields = get_all_incident_and_indicator_fields_from_id_set(id_set_file, 'mapper') + [
            field.lower() for field in BUILT_IN_FIELDS
        ] + LAYOUT_AND_MAPPER_BUILT_IN_FIELDS
```

Reply:
so in case a built-in field was being checked and it wasn't lower, we would miss that

Reply:
```
if any(simple.lower() == field.lower() for field in content_fields)
```
is also a good solution

Reply:
when is the case that we want to be case sensitive? using LAYOUT_AND_MAPPER_BUILT_IN_FIELDS only??

Reply:
no, also when getting the available incident fields from the id_set

Reply:
i am not sure why we use field.lower() when getting the `build_in_fields`, it was being used in the old implementation, so i kept it that way

Reply:
![image](https://user-images.githubusercontent.com/53861351/182629881-3ac7e551-6994-418c-bedc-159d4bda7b2d.png)
from old implementation